e2e-test/civiform_config.sh

```diff
@@ -0,0 +1,291 @@
+#! /usr/bin/env bash
```

Comment:
suggestion (blocking): change filename to reflect that this tests the aws_oidc template

Reply:
Changed to 'civiform_config_aws_oidc.sh'.